content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and one ethnicity is superior to all other ethnicities (racism).  
+
+or  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special. 
```

Comment:
The immediate response to that will be: "Yeah but they haven't taken over the world have they?" As a common core of the argument is might makes right. We need a cherry on top, something like "We should treasure and respect them for their unique attributes rather than crush them because we can" except... better :p

Reply:
Tried... in terms of the ecofeminism that follows....